src/cmd/singularity/cli/build.go

```diff
@@ -39,37 +39,34 @@ var buildflags = pflag.NewFlagSet("BuildFlags", pflag.ExitOnError)
 func init() {
 	BuildCmd.Flags().SetInterspersed(false)
 
-	BuildCmd.Flags().BoolVarP(&sandbox, "sandbox", "s", false, "Build image as sandbox format (chroot directory structure)")
+	BuildCmd.Flags().BoolVarP(&sandbox, "sandbox", "s", false, "build image as sandbox format (chroot directory structure)")
 	BuildCmd.Flags().SetAnnotation("sandbox", "envkey", []string{"SANDBOX"})
 
-	BuildCmd.Flags().StringSliceVar(&sections, "section", []string{"all"}, "Only run specific section(s) of deffile (setup, post, files, environment, test, labels, none)")
+	BuildCmd.Flags().StringSliceVar(&sections, "section", []string{"all"}, "only run specific section(s) of deffile (setup, post, files, environment, test, labels, none)")
 	BuildCmd.Flags().SetAnnotation("section", "envkey", []string{"SECTION"})
 
-	BuildCmd.Flags().BoolVar(&isJSON, "json", false, "Interpret build definition as JSON")
+	BuildCmd.Flags().BoolVar(&isJSON, "json", false, "interpret build definition as JSON")
 	BuildCmd.Flags().SetAnnotation("json", "envkey", []string{"JSON"})
 
-	BuildCmd.Flags().BoolVarP(&writable, "writable", "w", false, "Build image as writable (SIF with writable internal overlay)")
-	BuildCmd.Flags().SetAnnotation("writable", "envkey", []string{"WRITABLE"})
-
-	BuildCmd.Flags().BoolVarP(&force, "force", "F", false, "Delete and overwrite an image if it currently exists")
+	BuildCmd.Flags().BoolVarP(&force, "force", "F", false, "delete and overwrite an image if it currently exists")
 	BuildCmd.Flags().SetAnnotation("force", "envkey", []string{"FORCE"})
 
-	BuildCmd.Flags().BoolVarP(&update, "update", "u", false, "Run definition over existing container")
+	BuildCmd.Flags().BoolVarP(&update, "update", "u", false, "run definition over existing container (skips header)")
 	BuildCmd.Flags().SetAnnotation("update", "envkey", []string{"UPDATE"})
 
-	BuildCmd.Flags().BoolVarP(&noTest, "notest", "T", false, "Bootstrap without running tests in %test section")
+	BuildCmd.Flags().BoolVarP(&noTest, "notest", "T", false, "bootstrap without running tests in %test section")
```

Comment:
`"build without running tests in %test section"`

Reply:
nice catch